### PR TITLE
chore!: drop support multi-language

### LINKED ISSUE #115: 🆘 Drop multi-language support
## This task includes a small feature:
Image

**Kunlik hadis:**
...

### PR DESCRIPTION
**Pull request by toshmukhamedov**
Closes #115

### CODE REVIEW

File: translate/en.json [hunk 1]
```diff
@@ -26,7 +26,6 @@
   "fastingMessage": "Are you fasting?",
   "mainKeyboard": [
```
Comment (Xayrulloh) [inline translate/en.json:27]: why we need this file?

File: translate/ru.json [hunk 2]
```diff
@@ -26,7 +26,6 @@
   "fastingMessage": "Вы поститесь?",
   "mainKeyboard": [
     "🔍 Поиск",
```
Comment (Xayrulloh) [inline translate/ru.json:28]: same

File: types/database.ts [hunk 3]
```diff
@@ -1,40 +1,40 @@
-import { Document } from 'mongoose';
+import {Document} from 'mongoose'
 
 export interface IUser extends Document {
```
Comment (Xayrulloh) [inline types/database.ts:3]: db dan hammani language ni uzga to'g'illab qo'yiw kere

**Review by Xayrulloh — CHANGES_REQUESTED** [review]
1ta prettier run qivorin pls

File: translate/en.json (hunk 1 above)
Comment (toshmukhamedov) [reply translate/en.json:27]: We don’t need it anymore. I just wanted to keep your hard work)

Comment (toshmukhamedov) [reply translate/en.json:27]: I’ll delete it

File: types/database.ts (hunk 3 above)
Comment (toshmukhamedov) [reply types/database.ts:3]: dbdagi lang umuman ishlatilmidiku endi

File: translate/en.json (hunk 1 above)
Comment (Xayrulloh) [reply translate/en.json:27]: okay, don't worry my hard work stays on github history

File: types/database.ts (hunk 3 above)
Comment (Xayrulloh) [reply types/database.ts:3]: to'ri etasiz kalla qo'ydim, unaqada uyodan umuman o'cirvoriw kere

File: translate/en.json (hunk 1 above)
Comment (toshmukhamedov) [reply translate/en.json]: fixed

File: helper/language.ts [hunk 4]
```diff
@@ -1,12 +1,20 @@
 import fs from 'fs'
 import path from 'path'
+import {memoryStorage} from '#config/storage'
```
[the remark this answers is not in the document]
Comment (toshmukhamedov) [reply helper/language.ts]: fixed

File: translate/ru.json (hunk 2 above)
Comment (toshmukhamedov) [reply translate/ru.json]: fixed

File: types/database.ts (hunk 3 above)
Comment (toshmukhamedov) [reply types/database.ts:3]: fixed

File: types/context.ts [hunk 5]
```diff
@@ -1,8 +1,8 @@
 import { Context, SessionFlavor } from 'grammy'
-import { ScenesFlavor, ScenesSessionFlavor } from 'grammy-scenes'
+import { ScenesFlavor, ScenesSessionData } from 'grammy-scenes'
```
Comment (Xayrulloh) [inline types/context.ts:2]: whats difference between them

**Review by Xayrulloh — CHANGES_REQUESTED** [review]

File: types/context.ts (hunk 5 above)
Comment (toshmukhamedov) [reply types/context.ts:2]: SenenesSessionFlavor has been deprecated

**Review by eugene0928 — APPROVED** [review]

File: cron/cron.ts
Comment (eugene0928) [inline cron/cron.ts:1]: krilchada yozilganmidi hafta kunlari

Comment (toshmukhamedov) [reply cron/cron.ts:1]: lineni annotate qila olasizmi?

File: cron/cron.ts [hunk 6]
```diff
@@ -17,7 +17,7 @@ import { handleSendMessageError } from '#helper/errorHandler'
 async function monthly() {
   const now = new Date()
   const currentMonth = now.getMonth() + 1
-  const keyboardMessage = HLanguage('en', 'region')
+  const keyboardMessage = HLanguage('region')
   const regions = Object.keys(keyboardMessage)
   const regionIds = Object.values(keyboardMessage)
   const daysOfWeek = ['Якшанба', 'Душанба', 'Сешанба', 'Чоршанба', 'Пайшанба', 'Жума', 'Шанба']
```
Comment (toshmukhamedov) [inline cron/cron.ts:23]: @Xayrulloh I think @eugene0928 talks about this variable
Why is it in krill alphabet?

Comment (Xayrulloh) [reply cron/cron.ts:23]: Pdf ga oi

Comment (Xayrulloh) [reply cron/cron.ts:23]: Oid

**Review by eugene0928 — APPROVED** [review]

File: cron/cron.ts (hunk 6 above)
Comment (eugene0928) [reply cron/cron.ts:23]: are weekdays related to pdf?

Comment (toshmukhamedov) [reply cron/cron.ts:23]: yes

Comment (Xayrulloh) [reply cron/cron.ts:23]: Did u see entire code of that file.
I didn't that's why I've to look for it to answer your question.

Comment (Xayrulloh) [reply cron/cron.ts:23]: I easily forget the codes I wrote

Comment (eugene0928) [reply cron/cron.ts:23]: nope just u send on discord

Comment (Xayrulloh) [reply cron/cron.ts:23]: I recommend

**Pull request merged**